etc/canto-docker-initd

```diff
@@ -27,7 +27,7 @@ fi
 case "$action" in
   start)
     echo "Starting Canto with $WORKERS workers"
-    (date; cd $CANTO_SPACE; canto/script/canto_docker --non-interactive --use-container-name start_server --pid-file=/$PID_PATH --port $PORT -- script/canto_start --workers $WORKERS --keepalive-timeout 5 -s Starman --preload) >> canto.log 2>&1 &
+    (date; cd $CANTO_SPACE; canto/script/canto_docker --non-interactive --use-container-name start_server --pid-file=/$PID_PATH --port $PORT --signal-on-hup=QUIT --signal-on-term=QUIT -- script/canto_start --workers $WORKERS --keepalive-timeout 5 -s Starman) >> canto.log 2>&1 &
```

Comment:
I'm not sure we want `--signal-on-hup=QUIT` here, because the `HUP` signal is used to gracefully restart the server program, according to the docs for Server::Starter: "to gracefully restart the server program, send SIGHUP to the superdaemon." Starman also "supports HUP for graceful worker restarts".

In keeping with this behaviour, the `restart` command on the service script uses `HUP`:

```sh
  restart)
    pid=`/bin/cat $CANTO_SPACE/$PID_PATH`
    echo restarting $pid
    (cd $CANTO_SPACE; docker exec canto kill -HUP $pid)
    ;;
```
Changing the behaviour of the `HUP` signal to `QUIT` would presumably mean that a restart actually just stops the server and never starts it again.